tests/test_Imageclassification.py

```diff
@@ -0,0 +1,28 @@
+
+import mlflow
+import os
+from mlflow import pyfunc
+from datasets import load_dataset
+from transformers import ViTImageProcessor,ViTForImageClassification
+def vision_model_save():
+    Model_name="facebook/deit-base-patch16-224"
```

Comment:
```suggestion
    model_name="facebook/deit-base-patch16-224"
```
local variables should be lowercase